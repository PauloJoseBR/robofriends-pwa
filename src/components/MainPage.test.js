import { shallow } from 'enzyme';
import React from 'react';
import MainPage from './MainPage';

let wrapper;
beforeEach(() => {
    const mockProps = {
        onRequestRobots: jest.fn(),
        robots: [],
        searchFiield: '',
        isPending: false
    }

    wrapper = shallow(<MainPage {...mockProps} />)
})

it('renders MainPage without crash', () => {
    expect(wrapper).toMatchSnapshot();
})

it('it filters robots correctly', () => {
    expect(wrapper).toMatchSnapshot();
})