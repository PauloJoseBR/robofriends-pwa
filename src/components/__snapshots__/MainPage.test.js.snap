// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it filters robots correctly 1`] = `ShallowWrapper {}`;

exports[`renders MainPage without crash 1`] = `ShallowWrapper {}`;
